tests/test_apps/app_with_runtime_config_error/my_app.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from omegaconf import DictConfig
+
+import hydra
+
+
+def foo(cfg: DictConfig) -> None:
```

Comment:
I don't understand why ```def foo(cfg: DictConfig)``` is needed in this example. Doesnt that just make it more complicated? is it needed to create a more sophisticated stack trace?

Reply:
yes. I want to show that this shows all interesting stack frames for the user, not just the one with the error.